bin/fin

```diff
@@ -2476,12 +2476,23 @@ ssh_add () {
 	fi
 
 	local ssh_path="$HOME/.ssh"
-	# On Windows docker and docker-compose expect a Windows style path
-	is_windows && ssh_path=$(cygpath -w "$ssh_path")
+	# Check that ssh_path exists and is a directory, abort otherwise
+	if [[ ! -d "$ssh_path" ]]; then
+		echo-yellow "You don't seem to have any SSH keys in the default location (${ssh_path})."
+		_confirm "Would you like to generate a default SSH key pair now?"
```

Comment:
Needs to be:

```bash
_confirm "Would you like to generate a default SSH key pair now?" --no-exit
[[ "$?" != "0" ]] && return
# Generate an SSH key pair
echo "Generating a default SSH key pair..."
```

Otherwise `_confirm` will break the script with `exit 1`.

Reply:
Or maybe even simpler

```bash
_confirm "Would you like to generate a default SSH key pair now?" --no-exit || return
```